src/main/java/com/glucoseguardian/webbackend/application/adapter/LoginAdapter.java

```diff
@@ -0,0 +1,78 @@
+package com.glucoseguardian.webbackend.application.adapter;
+
+import com.glucoseguardian.webbackend.application.service.LoginService;
+import com.glucoseguardian.webbackend.storage.dto.LoginInputDto;
+import com.glucoseguardian.webbackend.storage.dto.LoginOutputDto;
+import com.glucoseguardian.webbackend.storage.dto.MessaggioDto;
+import com.glucoseguardian.webbackend.storage.dto.TotpDto;
+import com.glucoseguardian.webbackend.storage.entity.Admin;
+import com.glucoseguardian.webbackend.storage.entity.Dottore;
+import com.glucoseguardian.webbackend.storage.entity.Paziente;
+import com.glucoseguardian.webbackend.storage.entity.Tutore;
+
+public class LoginAdapter implements LoginAdapterInterface {
```

Comment:
Since the class handles login, logout, password change and I think in future registration (?), shouldn't be AuthAdapter a better name?